src/test/java/seedu/address/model/person/EmailTest.java

```diff
@@ -29,65 +29,62 @@ public void isValidEmail() {
         assertFalse(Email.isValidEmail(" ")); // spaces only
 
         // missing parts
-        assertFalse(Email.isValidEmail("@example.com")); // missing local part
-        assertFalse(Email.isValidEmail("peterjackexample.com")); // missing '@' symbol
+        assertFalse(Email.isValidEmail("@example.com")); // not the same as u.nus.edu
```

Comment:
This could have also been because you're missing local part. I think having 2 separate test cases for missing local part and not same as u.nus.edu might be better